src/main/java/com/manardenza/Main.java

```diff
@@ -8,6 +9,7 @@
 
     public static void main(String[] args) {
         System.out.println("Hello world!");
+        Injector injector = new Injector();
```

Comment:
Considering that all variables inside are static, you don't even need it's instance